src/Razor/src/Microsoft.VisualStudio.LanguageServer.ContainedLanguage/DefaultLSPRequestInvoker.cs

```diff
@@ -45,13 +45,13 @@ internal class DefaultLSPRequestInvoker : LSPRequestInvoker
             _serializer.AddVSExtensionConverters();
         }
 
-        public override Task<IEnumerable<TOut>> ReinvokeRequestOnMultipleServersAsync<TIn, TOut>(string method, string contentType, TIn parameters, CancellationToken cancellationToken)
+        public override Task<IEnumerable<(ILanguageClient LanguageClient, TOut Result)>> ReinvokeRequestOnMultipleServersAsync<TIn, TOut>(string method, string contentType, TIn parameters, CancellationToken cancellationToken)
```

Comment:
Instead of a struct could we build a record? I'm a hater of structs personally because they're a pain to pass around when writing new methods / become viral for consumers.

Reply:
Microsoft.VisualStudio.LanguageServer.ContainedLanguage is net471, which I don't believe supports C#9, or records. I could define it as a thin class if you'd prefer?

Reply:
We use the IsExternalInit package to get that in net472